django-tuttle/provider/management/commands/fetch_repositories.py

```diff
@@ -0,0 +1,21 @@
+from django.core.management.base import BaseCommand, CommandError
+from github import Github
+
+from provider.models import Repository, DeployKey
+from tuttleuser.models import TuttleUser
+
+
+class Command(BaseCommand):
+    help = 'command for get repositories list from github account'
+
+    def handle(self, *args, **options):
+        tuttle = TuttleUser.objects.get(username='girbons')
+
+        try:
+            login = Github(tuttle.token)
+        except:
```

Comment:
do not catch all the exceptions. look at the docs to see what exceptions the Github class can raise and catch only those.